google/ggrc-core
# GGRC-6669 Update template for control for LHN popup

# Issue description

Update UI of popup for control for LHN.

# Steps to test the changes

1. Activate LHN
2. Expand controls
3. Hover over any control

Expected result: control's popup has updated UI

# Solution description

Control's model flag was introduced previously. Based on this flag UI updates are applied.

# Sanity checklist



# PR Review checklist
